specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2020-07-01-preview/examples/CreateExtension.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "subscriptionId": "subId1",
+    "resourceGroupName": "rg1",
+    "clusterRp": "Microsoft.Kubernetes",
+    "clusterResourceName": "connectedClusters",
+    "extensionInstanceName": "ClusterMonitor",
+    "api-version": "2020-07-01-preview",
+    "clusterName": "clusterName1",
+    "extensionInstance": {
+      "properties": {
+        "extensionType": "azuremonitor-containers",
+        "autoUpgradeMinorVersion": true,
+        "releaseTrain": "Preview",
```

Comment:
This should be "Stable" since there is no azuremonitor-containers chart for "Preview" atm.